packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -203,38 +203,74 @@ class IconLabelButton extends StatelessWidget {
   }
 }
 
-class PauseButton extends IconLabelButton {
+class PauseButton extends StatelessWidget {
   const PauseButton({
-    Key? key,
-    double? minScreenWidthForTextBeforeScaling,
-    String tooltip = 'Pause',
-    required VoidCallback? onPressed,
-  }) : super(
-          key: key,
-          icon: Icons.pause,
-          label: 'Pause',
-          tooltip: tooltip,
-          minScreenWidthForTextBeforeScaling:
-              minScreenWidthForTextBeforeScaling,
-          onPressed: onPressed,
-        );
+    super.key,
+    this.iconOnly = false,
+    required this.tooltip,
```

Comment:
what's the reason of making this required instead of leaving as optional with a default value?

Reply:
We often forget to provide tooltip. If there is no tooltip, I want it to be visible explicitly in code reviews.